css-grid-3/Overview.bs

```diff
@@ -433,20 +433,24 @@ Intrinsic Tracks and repeat()</h4>
 	<div algorithm="determine intrinsic repetitions">
 		To determine the number of repetitions
 		that a ''repeat()'' function resolves to,
-		run layout as defined in [[#track-sizing-performance]],
-		with the following changes:
+		resolve the repeated track sizing functions to definite sizes
+		by initializing the track sizes (per [[css-grid-2#algo-init]])
+		and resolving intrinsic track sizes (per [[css-grid-2#algo-content]])
+		in accordance with [[#track-sizing]]
+		with the following assumptions:
 
 		* Expand ''repeat()/auto-fill''/''repeat()/auto-fit'' repeat functions once.
 		* Ignore explicit item placement.
 			(That is, assume all items have an [=automatic position=].)
+		* Do not [=collapsed grid track|collapse=] any tracks.
 		* If a [=masonry item=] has a span larger than 1,
-			then for each of its intrinsic sizes
-			that it would contribute to the [=virtual masonry item=],
+			then for each of its intrinsic size contributions,
```

Comment:
I still need to look into what this ends up implying, but seems reasonable to me